Src/BootCamp.Chapter/FileWithBalances.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class FileWithBalances
+    {
+        private readonly string _content;
+
+        public string GetContent()
+        {
+            return _content;
+        }
+
+        public FileWithBalances(string pathToFile)
+        {
+            if (String.IsNullOrEmpty(pathToFile)) throw new ArgumentException("Path to source file is empty");
+
+            string content = "";
+            try
+            {
+                content = File.ReadAllText(pathToFile);
+            }
+            catch (Exception ex)
+            {
+                throw new ArgumentException("Path to source file does not exist", ex);
+            }
+
+            if (String.IsNullOrEmpty(content))
```

Comment:
Please clarify the purpose of this. Looks redundant.

Reply:
The idea was to not do validation and/or cleaning if input is null (it's the first time checking if content is null/empty), but actually I think I have a better way to structure it